.vscode/settings.json

```diff
@@ -1,7 +1,26 @@
 {
-    "python.formatting.provider": "yapf",
-    "python.linting.pylintEnabled": false,
-    "python.linting.mypyEnabled": true,
+    "python.pythonPath": "/Users/cmcleod/conda/envs/ngsderive-dev/bin/python",
```

Comment:
This probably should be removed.

Reply:
If I remember right, VScode gets buggy if there isn't a `pythonPath` (multiple threads online about issues with this specifically and git. VScode doesn't seem to be addressing it).

I set up a local git function to replace and ignore that line on my machine, which was the best solution I found when I was making these changes.